src/forms/BoundDateField.test.tsx

```diff
@@ -14,12 +13,14 @@ describe("BoundDateField", () => {
     const r = await render(<BoundDateField field={author.birthday} onBlur={onBlur} onFocus={onFocus} />);
 
     // When focus is triggered on a checkbox
-    r.birthday().focus();
+    focus(r.birthday);
     // Then the callback should be triggered
     expect(onFocus).toBeCalledTimes(1);
 
     // When blur is triggered on a checkbox
-    r.birthday().blur();
+    blur(r.birthday());
+    // r.birthday().blur();
+    // });
```

Comment:
**nit:** Can we remove these commented out lines?

Reply:
Yep, done